### PR TITLE
[15.0][ADD] test mgmtsystem_nonconformity_quality_control_oca

### PR DESCRIPTION
**Pull request by mymage**

### CODE REVIEW

**Comment by mymage** [comment]
@ivantodorovich as promised, the tests

**Review by max3903 — APPROVED** [review]

File: mgmtsystem_nonconformity_quality_control_oca/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_quality_control_oca/tests/__init__.py]:
```suggestion
```

nitpicking: there's no need to put headers on __init__.py files

File: mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo.tests import common
+
+logger = logging.getLogger(__name__)
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py]:
```suggestion
from odoo.tests import common
```

Seems `logger` is not used

File: mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class TestQualityControl(common.TransactionCase):
+    def setUp(self):
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py]:
```suggestion
    def setUpClass(cls):
```
(and replace all occurrences of `self` with `cls` within this method)



Keep in mind `setUp` runs before every test.
whilst `setUpClass` runs only once for all tests, so it's more performant

File: setup/_metapackage/VERSION.txt [hunk 4]
```diff
@@ -1 +1 @@
-15.0.20221214.0
```
Comment (ivantodorovich) [inline setup/_metapackage/VERSION.txt]: What's this change? 🤔

File: setup/_metapackage/setup.py [hunk 5]
```diff
@@ -26,6 +26,7 @@
         'odoo-addon-mgmtsystem_nonconformity_hr>=15.0dev,<15.1dev',
         'odoo-addon-mgmtsystem_nonconformity_mrp>=15.0dev,<15.1dev',
         'odoo-addon-mgmtsystem_nonconformity_product>=15.0dev,<15.1dev',
+        'odoo-addon-mgmtsystem_nonconformity_quality_control_oca>=15.0dev,<15.1dev',
```
Comment (ivantodorovich) [inline setup/_metapackage/setup.py]: Same here. Weird. We shouldn't update this manually I think

File: mgmtsystem_nonconformity_quality_control_oca/tests/__init__.py (hunk 1 above)
Comment (mymage) [reply mgmtsystem_nonconformity_quality_control_oca/tests/__init__.py]: ok, copy that

File: mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py (hunk 2 above)
Comment (mymage) [reply mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py]: Yes, I forget to delete it

File: mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py (hunk 3 above)
Comment (mymage) [reply mgmtsystem_nonconformity_quality_control_oca/tests/test_inspection.py]: I tried but pre-commit dislike it. So I changed setUp but not cls.
Do you have some other suggestion?
Yes: add @classmethod before the method definition ;-)

File: setup/_metapackage/VERSION.txt (hunk 4 above)
Comment (mymage) [reply setup/_metapackage/VERSION.txt]: See bottom

File: setup/_metapackage/setup.py (hunk 5 above)
Comment (mymage) [reply setup/_metapackage/setup.py]: Probably I did something wrong. I did a rebase if this can fix things.

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**